x-pack/plugins/apm/server/routes/metrics/serverless/helper.test.ts

```diff
@@ -38,3 +42,85 @@ describe('calcMemoryUsedRate', () => {
     expect(calcMemoryUsedRate({ memoryFree: 50, memoryTotal: 100 })).toBe(0.5);
   });
 });
+
+const AWS_LAMBDA_PRICE_FACTOR = {
+  x86_64: 0.0000166667,
+  arm: 0.0000133334,
+};
+
+describe('calcEstimatedCost', () => {
+  it('returns 0 when price factor is not defined', () => {
+    expect(
+      calcEstimatedCost({
+        totalMemory: 1,
+        billedDuration: 1,
+        transactionThroughput: 1,
+        architecture: 'arm',
+      })
+    ).toEqual(0);
+  });
+
+  it('returns 0 when architecture is not defined', () => {
+    expect(
```

Comment:
same as above - it seems to me that a null-ish value is more appropriate here, because it allows the UI to distinguish between "could not compute" and "actual cost is 0 because there were no invocations"